acra/configuring-maintaining/optimisations/_index.md

```diff
@@ -2,3 +2,30 @@
 title: Optimisations
 bookCollapseSection: true
 ---
+
+[Vertical scaling]({{< ref "acra/configuring-maintaining/optimisations/vertical_scaling.md" >}})
+describes how performance will grow with more CPU cores / memory added
+
+[Horizontal scaling]({{< ref "acra/configuring-maintaining/optimisations/horizontal_scaling.md" >}})
+describes how performance will grow with instances being added
+
+[AcraStructs vs AcraBlocks]({{< ref "acra/configuring-maintaining/optimisations/acrastructs_vs_acrablocks.md" >}})
+describes difference between AcraStructs and AcraBlocks (two possible "containers" for encrypted database columns)
```

Comment:
not only database columns. AcraTranslator doesn't operates with columns, only with user data. Maybe just `encrypted data`?